test/WebJobs.Script.Tests.Integration/Management/FunctionsSyncManagerTests.cs

```diff
@@ -139,6 +139,7 @@ public FunctionsSyncManagerTests()
             _mockEnvironment.Setup(p => p.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteName)).Returns((string)null);
             _mockEnvironment.Setup(p => p.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteSlotName)).Returns((string)null);
             _mockEnvironment.Setup(p => p.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebJobsFeatureFlags)).Returns((string)null);
+            _mockEnvironment.Setup(p => p.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteSku)).Returns((string)ScriptConstants.DynamicSku);
```

Comment:
Should there be a line like this but for ScriptConstants.FlexConsumptionSku as well?

Reply:
This is just the default for all the existing tests, when not explicitly overridden by individual tests. I had to add this because the sync codepaths now check sku where they didn't before, so a mock had to be set up.

I have added some Flex specific tests to verify the new logic - see below.

Reply:
I see, thanks for clarifying!